tests/filtered_relation/tests.py

```diff
@@ -60,7 +60,7 @@ def test_filtered_relation_alias_mapping_other(self):
                         'book', alias='book_alice',
                         condition=Q(book__title__iexact='poem by alice'))
                     .filter(book_alice__isnull=False))
-        self.assertIn('INNER JOIN "filtered_relation_book" book_alice ON', str(queryset.query))
```

Comment:
I'm not sure about this change. We need to force aliasing on SQL level I believe. Otherwise we don't need to accept `alias` argument.